embed-gutenberg-block-google-maps.php

```diff
@@ -128,7 +128,7 @@ function renderGutenbergMapEmbedblock( $attributes ) {
     $output .= '</div></div>';
 
     // Return the output
-    return $output;
+    return apply_filters('render_pantheon_google_map_block', $output);
```

Comment:
I would think the filter needs `$attributes` passed as the `$args` parameter like this: `return apply_filters('render_pantheon_google_map_block', $output, $attributes);` so that the block attributes are available to the hooked function.

Can you post an example of your function that makes use of the filter?

Reply:
Sure I can:

```php
add_filter('render_pantheon_google_map_block', function($content) {
    return '<div class="m-page-wrapper"><div class="row">' . $content . '</div></div>';
});
```

I'm posting requested change afterwards.

Reply:
Ah, I see you are just adding your own wrapping div, not modifying the generated HTML completely. I think the attributes will help people who want to totally customize the output as well.